tools/test-npm.sh

```diff
@@ -37,7 +38,7 @@ node cli.js rebuild
 # install npm devDependencies and run npm's tests
 node cli.js install --ignore-scripts
 # run the tests
-node cli.js run-script test-node
+node cli.js run-script test-node -- --reporter=tap | tee ../test-npm.tap
```

Comment:
It might be worth trying to replicate the current api exposed by the python test runner. An option to specify reporter, and an option to specify output file name `-p tap --logfile test.tap`